### PR TITLE
New package: xfce4-clipman-plugin-1.2.6

### PR DESCRIPTION
**Pull request by aadityabagga**

### CODE REVIEW

File: srcpkgs/xfce4-clipman-plugin/INSTALL [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# INSTALL
```
Comment (xtraeme) [inline srcpkgs/xfce4-clipman-plugin/INSTALL]: This INSTALL script is completely wrong.

xbps-src will add this automagically through `xbps-triggers`.

The only requirement is just adding:

```
depends="gtk-update-icon-cache"
```

File: srcpkgs/xfce4-clipman-plugin/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+revision=1
+build_style=gnu-configure
+makedepends="gtk+-devel xfce4-panel-devel libunique1-devel libXtst-devel qrencode-devel exo-devel libxfce4ui-devel"
+hostmakedepends="intltool pkg-config"
+short_desc="A clipboard plugin for the Xfce4 panel"
+maintainer="Aaditya Bagga <abchk1234@gmail.com>"
+license="GPL-2"
+homepage="http://goodies.xfce.org/projects/panel-plugins/xfce4-clipman-plugin"
+distfiles="http://archive.xfce.org/src/panel-plugins/xfce4-clipman-plugin/1.2/xfce4-clipman-plugin-${version}.tar.bz2"
+checksum="23043f99c3e5257d1f3d68b5ee5125e3469c15620d098e22c6250386197f48a5"
```
Comment (xtraeme) [inline srcpkgs/xfce4-clipman-plugin/template]: Missing newline at EOF

**Comment by xtraeme** [comment]
Looks good this time, thanks :+1:

**Comment by xtraeme** [comment]
Please use the commit guidelines explained in `CONTRIBUTING.md` for your next pull requests.

Cheers

**Comment by aadityabagga** [comment]
Sure, thx for the help.

**Pull request merged**